apps/src/javalab/JavabuilderConnection.js

```diff
@@ -59,7 +58,7 @@ export default class JavabuilderConnection {
         levelId: this.levelId,
         options: this.options,
         executionType: this.executionType,
-        contentManagerType: ContentManagerType.DASHBOARD,
+        useContentManager: false,
```

Comment:
Since ContentManager is a Javabuilder implementation detail, it'd be better to name this with the details that Java Lab knows about. For example, Java Lab knows whether it's pushed the sources to Javabuilder already or if Javabuilder should get the sources from dashboard. Something like `useDashboardSources: true` or `usePushedSources: false` would better reflect the separation of concerns.

Reply:
Makes sense. Naming things is hard :). Give it another look!